source/main/modules/window-manager/index.ts

```diff
@@ -587,10 +657,19 @@ export default class WindowManager {
   /**
    * Shows the preferences window
    */
-  showPreferences (): void {
+  showPreferences(): void {
     if (this._preferences === null) {
-      const conf = this._retrieveWindowPosition('preferences', null)
-      this._preferences = createPreferencesWindow(conf)
+      // const conf = this._retrieveWindowPosition('preferences', null)
+      let conf: WindowPosition = {
+        windowType: 'preferences',
+        lastDisplayId: 69734662,
+        top: 279.5,
+        left: 448,
+        width: 896,
+        height: 513,
+        isMaximised: false
+      }
+      this._preferences = createPreferencesWindow(conf);
```

Comment:
Don't change the size of the window because changing the window size isn't part of the linked issues.